test/onemkl.jl

```diff
@@ -14,5 +14,9 @@ k = 13
         B = oneArray{T}(undef, m)
         oneMKL.copy!(m,A,B)
         @test Array(A) == Array(B)
+		
+		# Test axpy primitive
+		alpha = rand(T,1)
+		@test testf(axpy!, alpha[1], rand(T,m), rand(T,m))
```

Comment:
Could you wrap these in a `@testset "axpy"`? Maybe put the ones above in a `@testset "copy"` too. This is to avoid dependencies between these tests; we wouldn't want, e.g., the `A` or `B` arrays to be reused, because that makes it hard to refactor tests later.

Reply:
@maleadt  agree, sure will do that. Thanks

Reply:
@maleadt done please take a look